doc/instruction.txt

```diff
@@ -104,11 +38,6 @@ Running the application (Windows).
                 0 5 0
             This means that a clique consisting of two vertices (with indices 1 and 2) was found.
             The part of the adjacency matrix corresponding to the clique is colored red.
-        
-        Run the heuristic maximum clique finding algorithm on the C125.9 DIMACS benchmark instance with default size definition (vertex count)
```

Comment:
Can we include it with some info that this is an additional functionality described in the report?

Reply:
DIMACS is already described. I don't think he will even look at it